src/machines/java/com/enderio/machines/client/gui/widget/ExperienceWidget.java

```diff
@@ -18,43 +17,35 @@ public class ExperienceWidget extends EIOWidget {
     private static final ResourceLocation GUI_ICONS_LOCATION = new ResourceLocation("textures/gui/icons.png");
     private final Screen displayOn;
     private final Supplier<FluidTank> getFluid;
-    private final Supplier<Integer> maxXP;
 
-    public ExperienceWidget(Screen displayOn, Supplier<FluidTank> getFluid, Supplier<Integer> maxXP, int pX, int pY, int pWidth, int pHeight) {
```

Comment:
I think maxXP needs to stay, as it is used to display the amount exp needed for the crafting, not the amount stored. However, that system is broken and needs to be fixed, see #363 .

Reply:
That's a different Widget entirely. I have renamed the old widget to `ExperienceCraftingWidget`. This new one only shows fluid to XP level along with progress